examples/pipelines/controlnet-interior-design/run_locally.sh

```diff
@@ -0,0 +1,19 @@
+export METADATA='{"run_id":"test_niels", "base_path":"/Users/nielsrogge/Documents/fondant_artifacts"}'
```

Comment:
I would not merge this into main but it is a nice script we can start improving to speed up local development